src/devices/src/lib.rs

```diff
@@ -33,4 +34,6 @@ pub enum Error {
     IoError(io::Error),
     /// Device received malformed payload.
     MalformedPayload,
+    /// Error during queue processing.
+    QueueError(QueueError),
```

Comment:
nit: we try to keep `enum` variants alphabetically ordered (to be easy to find/navigate)

Reply:
Solved